packages/dd-trace/src/priority_sampler.js

```diff
@@ -39,12 +39,28 @@ const DEFAULT_KEY = 'service:,env:'
 
 const defaultSampler = new Sampler(AUTO_KEEP)
 
+/**
+ * from config.js
+ * @typedef { sampleRate: number, provenance: string, rateLimit: number, rules: SamplingRule[] } SamplingConfig
+ *
+ * empirically defined
+ * @typedef {2|-1|1|0} SamplingPriority
+ */
 class PrioritySampler {
+  /**
+   * @param env {string}
```

Comment:
Nit: Would you mind renaming this to `environment`? I feel we could generally use less short versions to make things more explicit :)

Reply:
Usually I'd agree, but `env` is a standard across all of Datadog ([src](https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/?tab=kubernetes)) so it's more understandable in its short-form